File: Linked-Lists/C++/introduction to linked list.cpp
#include <vector>
#include <iostream>
using namespace std;
class Node{
    public:
    int data;
    Node* next;
    public:
    Node(int data1,Node* next1)
    {
        data=data1;
        next=next1;
    }
    public:
    Node(int data1)
    {
        data=data1;
        next=nullptr;
    }
};
int main() {
    // Write C++ code here
    vector<int>arr={2,5,8,7};
    Node* y=new Node(arr[0]);
    cout<<y->data;

    return 0;
}

File: Linked-Lists/Java/Introduction_to_Linked_List.java
class Node {

    int data;
    Node next;

    Node(int data1, Node next1) {
        data = data1;
        next = next1;
    }

    Node(int data1) {
        data = data1;
        next = null;
    }

};


public class Introduction_to_Linked_List {
    public static void main(String[] args) {


        int[] arr = {1, 2, 3, 4, 5};
        Node y = new Node(arr[1]);
        System.out.println(y.data);
    }
}
